src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -106,9 +106,7 @@
 
         failedItemsData.forEach(({ contentName, contentTypeName }) => {
             const container = doc.createElement('tbody');
-            const renderedItem = rowTemplate
-                .replace('{{ content_name }}', contentName)
-                .replace('{{ content_type_name }}', contentTypeName);
+            const renderedItem = rowTemplate.replace('{{ content_name }}', contentName).replace('{{ content_type_name }}', contentTypeName);
```

Comment:
That's not `renderedItem`, more like `filledRow` , as you don't render here anything.

But you haven't touched here anything except formatting, so